lib/api/model/events.dart

```diff
@@ -766,14 +767,6 @@ class UpdateMessageEvent extends Event {
   Map<String, dynamic> toJson() => _$UpdateMessageEventToJson(this);
 }
 
-/// As in [UpdateMessageEvent.propagateMode].
-@JsonEnum(fieldRename: FieldRename.snake)
-enum PropagateMode {
```

Comment:
And this can move from lib/api/model/events.dart to lib/api/model/model.dart, since it's no longer specific to an event.